addons/crm/models/crm_team_member.py

```diff
@@ -174,9 +176,7 @@ def _assign_and_convert_leads(self, work_days=2):
                     user_ids=member_data['team_member'].user_id.ids
                 )
 
-                # auto-commit except in testing mode
-                auto_commit = not getattr(threading.currentThread(), 'testing', False)
-                if auto_commit:
+                if auto_commit and counter % 100 == 0:
```

Comment:
Couldn't we allow its tuning through a config parameter (using 100 as default value if not set) ? We already allow crm.assignment.delay and crm.assignment.bundle, we could have crm.assignment.commit.bundle . That way if we have to tweak it on some db it is easy to do.